src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/SignatureDescription.cs

```diff
@@ -27,21 +29,26 @@ public SignatureDescription(SecurityElement el)
 
         public virtual AsymmetricSignatureDeformatter CreateDeformatter(AsymmetricAlgorithm key)
         {
-            AsymmetricSignatureDeformatter item = (AsymmetricSignatureDeformatter)CryptoConfig.CreateFromName(DeformatterAlgorithm);
+            Debug.Assert(DeformatterAlgorithm != null);
+            AsymmetricSignatureDeformatter? item = (AsymmetricSignatureDeformatter?)CryptoConfig.CreateFromName(DeformatterAlgorithm);
+            Debug.Assert(item != null);
```

Comment:
These asserts aren't valid... `new SignatureDescription().CreateDeformatter()` is going to have a null deformatter algorithm; and if they ask for a non-existing item from CryptoConfig, that'll be null, too.

I'd just ! them away and maintain compatibility... but we shouldn't assert when it's possible to trip the asserts.